runtime/README.md

```diff
@@ -246,10 +252,6 @@ A Runtime Firmware update is triggered by the `CALIPTRA_FW_LOAD` command. Upon
 receiving this command, Runtime Firmware will:
 
 1. Write-lock mailbox
-1. Verify firmware image signature
-    1. Preserve hash for later steps
-1. Update the “Latest TCI” field of the TCI Node which contains the Runtime
-   Journey PCR (TYPE = RTJM, “Internal TCI” flag is set) with hash.
 1. Invoke “Impactless Reset”
 
 Once Impactless Reset has been invoked, FMC will load the hash of the image
```

Comment:
does this imply that the hash will not be copied if the Manifest does not pass verification?

Reply:
Hmm. So we have 2 PCRs here that get extended by FMC:

1. RT Journey
2. RT Current

I think the logic has to be:

* If verification passes, ROM resets (2), FMC extends both (1) and (2)
* If verification fails, ROM doesn't reset (2), FMC does nothing

cc @rusty1968

Reply:
Added these details